src/test/java/io/github/acm19/aws/interceptor/http/AwsRequestSigningApacheInterceptorTest.java

```diff
@@ -216,6 +239,11 @@ private static int getContentLength(final InputStream content) {
                 return -1;
             }
         }
+
+        @Override
+        public CompletableFuture<AsyncSignedRequest> signAsync(AsyncSignRequest asyncSignRequest) {
```

Comment:
There's a default implementation, why are you overriding it?

Reply:
`HttpSigner` is an interface? Unless I'm missing something.